sql/sqlserver/server.go

```diff
@@ -100,7 +105,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Send the Request for SQL execution.
-	s.send(sqlwire.Call{Args: args, Reply: reply})
+	s.execute(sqlwire.Call{Args: args, Reply: reply})
```

Comment:
We call this `exec` almost everywhere else. Not sure why it would be spelled out here.